README.md

```diff
@@ -39,7 +39,64 @@ use {
 Call or map this command
 
 ```
-:lua require("telescope").extensions.live_grep_raw.live_grep_raw()
+:lua require('telescope').extensions.live_grep_raw.live_grep_raw()
+```
+
+You can also call it passing arguments similar to `builtin.live_grep`
+
+```
+:lua require('telescope').extensions.live_grep_raw.live_grep_raw({vimgrep_arguments={ ... }})
+```
+
+Note that for `vimgrep_arguments` some `rg` options are required for the output to be in the right format to parse. Below are the minimal recommended and default options. Check `:help vimgrep_arguments` for details
+
+```lua
+vimgrep_arguments = {
```

Comment:
I would remove this or link to the telescope README/code?  
Because if the vimgrep args in telescope change there is a possibility that they are no longer in sync.

Reply:
Sure, I was gonna ask about it. I'll remove it.